ddtrace/pin.py

```diff
@@ -45,7 +52,9 @@ def get_from(obj):
         """
         if hasattr(obj, '__getddpin__'):
             return obj.__getddpin__()
-        return getattr(obj, '_datadog_pin', None)
+
+        pin_name = _DD_PIN_PROXY_NAME if isinstance(obj, wrapt.ObjectProxy) else _DD_PIN_NAME
```

Comment:
While I think it's correct, in which integration does it happen? does it create issues with the current instrumentation if the `_datadog_pin` is propagated to the wrapped object? just asking to know what this could lead to.